rebasehelper/tests/test_specfile.py

```diff
@@ -199,6 +200,97 @@ def test_find_archive_target_in_prep(self, spec_object):
         target = spec_object.find_archive_target_in_prep('misc.zip')
         assert target == 'test-1.0.2/misc'
 
+    @pytest.mark.parametrize('spec_attributes, kwargs, expected_content', [
+        (
+            {
+                'save': lambda: None,
+                'removed_patches': [],
+                'spec_content': dedent("""
```

Comment:
I think there should be a backslash at the end of this line. An empty line perhaps doesn't matter here, but it could matter in some other test, and this can't serve as a template in such case.